boa/integrations/jupyter/browser.py

```diff
@@ -134,16 +134,17 @@ def __init__(self, address=None):
         self.signer = BrowserSigner(address)
         self.set_eoa(self.signer)
 
-    def get_chain_id(self):
-        return _javascript_call(
+    def get_chain_id(self) -> int:
+        chain_id = _javascript_call(
             "rpc", "eth_chainId", timeout_message=RPC_TIMEOUT_MESSAGE
         )
+        return int.from_bytes(bytes.fromhex(chain_id[2:]), "big")
```

Comment:
`int(chain_id[2:], 16)`

but are they really expecting an int to come back? i would expect a hex string

Reply:
@bout3fiddy ?

Reply:
> fiddyresearch:
It’s always a hex string coming back

> Daniel Schiavini
Is that what you would want?

> fiddyresearch:
Not. Preferably integer

> fiddyresearch:
That’s a better devex